modules/apps/asset/asset-categories-test/src/testIntegration/java/com/liferay/asset/categories/service/test/AssetCategoryServiceTest.java

```diff
@@ -70,6 +72,66 @@ public void setUp() throws Exception {
 			ServiceContextTestUtil.getServiceContext());
 	}
 
+	@Test
+	public void testAddCategoryWithGuestViewPermission() throws Exception {
```

Comment:
@AliciaGarciaGarcia, why not create a private method called _testAddCategoryWithViewPermission and pass the RoleConstant?

Reply:
no strong opinion.

I'll rebase this and resend :D